test/e2e/checkpoint_test.go

```diff
@@ -1285,6 +1285,7 @@ var _ = Describe("Podman checkpoint", func() {
 
 		// As the container has been started with '--rm' it will be completely
 		// cleaned up after checkpointing.
+		// Cannot use ExitCleanly() because "skipping [ssh-agent-path] since it is a socket"
```

Comment:
This is the one case that failed on my laptop when I tried `ExitCleanly()`. I didn't take the time to figure out why.